reqs/build.pip

```diff
@@ -1,4 +1,4 @@
-numpy<1.20
+numpy<1.20; platform_machine != "arm64"
```

Comment:
Can we handle both cases, in this file, without making the change in `env_create.sh` ? 
```
numpy<1.20; platform_machine != "arm64"
numpy==1.19.5; platform_machine == "arm64"
```

Reply:
I don't think so. The numpy version we need can only be installed via `conda` not `pip`.